CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Views/NotificationPopupPage.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using CheckDrive.Mobile.ViewModels;
+using Rg.Plugins.Popup.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace CheckDrive.Mobile.Views
```

Comment:
here too  ;  It would be good to write down

Reply:
Xamarin forms doesn't support this technology